pkg/scheduler/plugins/elasticquota/core/group_quota_manager.go

```diff
@@ -145,6 +145,10 @@ func (gqm *GroupQuotaManager) updateGroupDeltaRequestTopoRecursiveNoLock(deltaRe
 		newSubLimitReq := curQuotaInfo.getLimitRequestNoLock()
 		deltaReq = quotav1.Subtract(newSubLimitReq, oldSubLimitReq)
 
+		if curQuotaInfo.Name == extension.SystemQuotaName || curQuotaInfo.Name == extension.DefaultQuotaName {
```

Comment:
as we talked,  if the curQuotaInfo is one of (extension.SystemQuotaName, extension.DefaultQuotaName),  we should update the requested of Quota. And refactor the function, make the function clear.

Reply:
fixed